keras_nlp/models/bert/bert_presets.py

```diff
@@ -12,13 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from collections import defaultdict
-
-# TODO(jbischof): document checkpoints in keras.io and use URL in docstrings
+# TODO(jbischof): document presets in keras.io and use URL in docstrings
 # Metadata for loading pretrained model weights.
-checkpoints = {
+presets = {
     "bert_tiny_uncased_en": {
-        "model": "BertTiny",
+        "config": {
+            "vocabulary_size": 30522,
+            "num_layers": 2,
+            "num_heads": 2,
+            "hidden_dim": 128,
+            "intermediate_dim": 512,
+            "dropout": 0.1,
+            "max_sequence_length": 512,
+            "num_segments": 2,
+        },
         "vocabulary": "uncased_en",
```

Comment:
I think we should probably flatten the vocabularies into the persets too with this change and add `from_preset` to `BertPreprocessor`. But we can do that in a follow up.

Reply:
Yes, I left to TODO for that. I'm not sure I love the idea of everything flattened since it seems difficult to maintain, but let's punt on that discussion for now.